en/docs/install-and-setup/setup/deployment-best-practices/tuning-performance.md

```diff
@@ -419,3 +419,41 @@ The registry indexing process, which indexes the APIs in the Registry, is only r
 [indexing]
 enable = false
 ```
+
+### Optimizing database indexing for case-sensitive and case-insensitive user stores
+
+One key aspect of performance optimization is database indexing. While API Manager includes essential database indexes in its default database scripts, certain scenarios may require additional indexing. Database Administrators (DBAs) are responsible for implementing these additional indexes to ensure optimal performance.
+
+For user stores configured with case-insensitive usernames in databases such as Oracle, PostgreSQL, or SQL Server, the DBA should create the necessary indexes using the `LOWER(UM_USER_NAME)` function.
+
+If a case-sensitive database is in use, only the indexes listed below should be added, and configurations specified beyond the third point should be disregarded. Note that these indexes may not cover all possible indexing needs.
```

Comment:
Mentioning `third point` would be confusing as we don't have a specific 3rd point here.